Google and Github OAuth

jonas-kgomo opened the pull request:
Login for google and github users , accounts should be different.

jonas-kgomo committed:
adding supabase ui

jonas-kgomo committed:
Merge branch 'develop' into test

jonas-kgomo committed:
github and google login oauth

chalkedgoose commented:
Is there any chance we could get a preview of what this PR does a Gif or MP4?

coderinblack08 commented:
He sent a mp4 to me, works great

coderinblack08 commented:
Lol why did you modify the footer and add `@supabase/ui`, seems totally unrelated

jonas-kgomo commented:
after our discussion about using our design system I removed superbase-ui.  About footer,  i noticed the footer wasn't showing in `develop` branch on `components/layout`, so i added that as well (after @chalkedgoose
suggested)

chalkedgoose commented on pages/index.tsx:
```diff
@@ -30,8 +33,14 @@ const Index: React.FC = () => {
-          <Button>Or Join with GitHub</Button>
+          <Button
+            onClick={async () =>
+            { await supabase.auth.signIn({ provider: 'google' }) }}
```
Is there a reason for using async await in your onClick callback?
wont the browser execute the promise anyways?

jonas-kgomo replied:
Correct,  `await` makes sense for `signout`, not sign in, I will remove this

chalkedgoose commented on pages/index.tsx:
```diff
@@ -30,8 +33,14 @@ const Index: React.FC = () => {
+            { await supabase.auth.signIn({ provider: 'google' }) }}
+          > Join with Google</Button>
+          <Button
+           onClick={async () =>
```
Same here.

chalkedgoose commented:
Rebasing your PR should get rid of conflict
